### PR TITLE
Merge paramlist into namelist

### PR DESCRIPTION
**Pull request by charleskawczynski**
@yairchn suggested we merge the namelist and paramlist. @haakon-e, I believe this will affect the calibration interface.

### CODE REVIEW

**Comment by szy21** [comment]
I think this is a good idea. I may call it "namelist" rather than "paramlist" because it's not just parameters. I don't have a strong preference though.

**Review by szy21 — APPROVED** [review]

**Review by yairchn — APPROVED** [review]
LGTM - I am joining Zhaoyi's comment that namelist is a better name for the combined file

**Comment by charleskawczynski** [comment]
Fine by me, I'll do a replace-all to "namelist"

File: src/NetCDFIO.jl [hunk 1]
```diff
@@ -57,7 +57,7 @@ mutable struct NetCDFIO_Stats
         open(joinpath(outpath, "$simname.in"), "w") do io
             JSON.print(io, namelist, 4)
         end
```
Comment (haakon-e) [inline src/NetCDFIO.jl:59]:
```suggestion
```
remove this and change the comment

**Review by haakon-e — APPROVED** [review]
Looks good. Merge and I'll fix things on the calibration side.

**Pull request merged**